packages/common

```diff
@@ -25,8 +25,9 @@ shotwell
 slick-greeter
 system-config-printer
 unzip
-vlc
 webrtc-audio-processing
 xconfig
 xorg-minimal
 zip
+ly
+mpv
```

Comment:
Changing to MPV should be a request ticket for the community to vote on. 

 I do not want Ly to be part of the default installation.

Reply:
To be clear, the more packages that get added, the more RAM the system needs to run in live.